Automattic/course
# Hide font settings on footer site title

### Introduced change
- Create a site title variation
- Add a filter to hide the Typography and Dimensions settings panel only when the variation is selected

### Testing instructions
- Open the footer template part  that is using the site title
- Select the site title
- Check if the Typography and Dimensions settings panel are not visible

<img width="1662" alt="Screenshot 2022-12-02 at 19 21 42" src="https://user-images.githubusercontent.com/38718/205399924-e875a3e8-f13e-4c4b-aa59-60a96dd96bf0.png">


### Extra comments:
Variation name suggestions will be very well received 😄
